src/v2/Apps/ArtistSeries/__tests__/ArtistSeriesApp.jest.tsx

```diff
@@ -55,31 +57,34 @@ describe("ArtistSeriesApp", () => {
         const wrapper = await getWrapper()
         expect(wrapper.find("AppContainer").length).toBe(1)
         expect(wrapper.find("ArtistSeriesHeader").length).toBe(1)
+        expect(wrapper.find("ArtistSeriesArtworksFilter").length).toBe(1)
+      })
+
+      describe("ArtistSeriesArtworksFilter", () => {
+        it("renders correctly", async () => {
+          const wrapper = await getWrapper()
+          expect(wrapper.find("ArtworkGrid").length).toBe(2)
```

Comment:
#minor but I would expect there to be just one `ArtworkGrid`! (Are there 2 due to multiple breakpoints or something?)

Checking for `GridItem` might be more accurate as that should be counting the number of artworks that appear.

Reply:
I dug into the multiple `ArtworkGrid`s and turns out we have two _nested_ `ArtworkGrid`s 🤔  Unclear why that is, but decided to check for `ArtworkFilterArtworkGrid` instead since it effectively tests for the same thing.